### PR TITLE
FORMS-13981 - Encode formDefinition with jsonString context

### PR DESCRIPTION
**Pull request by TalmizAhmed**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/page/.content.xml [hunk 1]
```diff
@@ -3,5 +3,5 @@
     jcr:description="Page Component for the Forms Components Library"
     jcr:primaryType="cq:Component"
     jcr:title="Page"
-    sling:resourceSuperType="core/fd/components/page/v1/page"
+    sling:resourceSuperType="core/fd/components/page/v2/page"
```
Comment (nit23uec) [inline examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/page/.content.xml:6]: is this change related to the bug we are addressing?

**Comment by nit23uec** [comment]
pls avoid referencing internal jira ids in open source.

File: examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/page/.content.xml (hunk 1 above)
Comment (TalmizAhmed) [reply examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/page/.content.xml:6]: Yes, and it breaks without this change. The main tag's contents are now inside of the body.html, which is a part of v2/page ([see here](https://github.com/adobe/aem-core-forms-components/pull/1194/files#diff-27af9ae2f33f90df3524749ed8dc9a6f3f77422c70494d31c2c07706eb2476b7)). I believe this was a miss in that PR.
Without this being updated in examples, edge rendition would not receive the json.

**Comment by TalmizAhmed** [comment]
@nit23uec Removed jira Id from commit message
